config/default.yml

```diff
@@ -9,6 +9,87 @@ Bundler/DuplicatedGem:
 Bundler/OrderedGems:
   Enabled: true
```

Comment:
Do you like this? I tend to disable it. I do generally sort gems now, but sometimes I like to keep things in logical groupings.

Reply:
tl;dr: IDK

I included this from the parent's config and haven't considered this one individually before now. Optimistically, I could see a predictable ordering making life easier for troubleshooting or a developer that's new to a project. 

I also like to keep things grouped logically with supporting comments, but it's always been a struggle to maintain that over time. When a gem gets stuck into one of the groupings without thought I'm more likely to overlook it later when troubleshooting.